Reference_Manual.md

```diff
@@ -3287,6 +3294,245 @@ This typed_data allows you to retrieve the data type of the underlying binary to
 3. <b>identified_data_serializer serialized objects</b>: `factory_id`, `class_id` and `type_id` are non-negative values.
 4. <b>Portable serialized objects</b>: `factory_id`, `class_id` and `type_id` are non-negative values.
 5. <b>Custom serialized objects</b>: `factory_id=-1`, `class_id=-1`, `type_id` is the non-negative type ID. 
+## 7.11. SQL Module
+To use this service, the Jet engine must be enabled on the members and the `hazelcast-sql` module must be in the classpath of the members.
+If you are using the CLI, Docker image, or distributions to start Hazelcast members, then you don't need to do anything, as the above preconditions are already satisfied for such members.
+
+However, if you are using Hazelcast members in the embedded mode, or receiving errors saying that The Jet engine is disabled or `Cannot execute SQL query because "hazelcast-sql" module is not in the classpath.` while executing queries, enable the Jet engine following one of the instructions pointed out in the error message, or add the `hazelcast-sql` module to your member's classpath.
```

Comment:
Can we provide link to the reference manual sql section for server side details?

Reply:
Fixed.

Reply:
I meant the Ref man [here](https://docs.hazelcast.com/hazelcast/latest/getting-started/install-hazelcast)

Reply:
Fixed.